src/client/src/containers/PrivacyModal/index.tsx

```diff
@@ -0,0 +1,127 @@
+import classnames from "classnames";
+import * as React from "react";
+import { connect } from "react-redux";
+
+import asModal from "../../components/Modal";
+import buttonStyles from "../../css/buttonStyles.module.css";
+import styles from "./styles.module.css";
+import { ReactComponent as Cancel } from "../../assets/cancel.svg";
+import { isPrivacyModalOpenSelector } from "../../selectors/modalSelector";
+
+import { defineMessages, injectIntl, InjectedIntlProps } from "react-intl";
+import { closeModalAction } from "../../actions/modalActions/modalActions";
+import { AppState } from "../../reducers";
+import { Dispatch } from "redux";
+import RootAction from "../../actions/ActionType";
+
+import { IDependency } from "../DependencyInfo";
+
+interface IStateProps {
+  isModalOpen: boolean;
+  dependency: IDependency;
+}
+
+interface IDispatchProps {
+  closeModal: () => any;
+}
+
+type Props = IStateProps & IDispatchProps & InjectedIntlProps;
+
+const messages = defineMessages({
+  beingRedirected: {
+    id: "privacyModal.beingRedirected",
+    defaultMessage: "You are being redirected."
+  },
+  thirdPartyWebsite: {
+    id: "privacyModal.thirdPartyWebsite",
+    defaultMessage:
+      "You will be taken to the {thirdPartyWebsite} which is a non-Microsoft service."
+  },
+  privacyStatement: {
+    id: "privacyModal.privacyStatement",
+    defaultMessage: "Privacy Statement"
+  },
+  OK: {
+    id: "privacyModal.OK",
+    defaultMessage: "OK"
+  }
+});
+
+const PrivacyModal = (props: Props) => {
+  const { dependency, intl } = props;
+
+  if (dependency === null || dependency === undefined) {
+    return null;
+  }
+
+  const { downloadLink, privacyStatementLink, downloadLinkLabel } = dependency;
+
+  const cancelKeyDownHandler = (event: React.KeyboardEvent<SVGSVGElement>) => {
+    if (event.keyCode === 13 || event.keyCode === 32) {
+      event.preventDefault();
+      event.stopPropagation();
+      props.closeModal();
+    }
+  };
+
+  return (
+    <div>
+      <div className={styles.headerContainer}>
+        <div className={styles.title}>
+          {intl.formatMessage(messages.beingRedirected)}
+        </div>
+        <Cancel
+          tabIndex={0}
```

Comment:
Does tabIndex=0 give us something here?

Reply:
Yes, it lets you navigate using tab onto the X button on the top right